packages/fuels-abigen-macro/tests/test_projects/enum_inside_struct/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["lulwat"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "enum_inside_struct"
+
+[dependencies]
+std = { git = "https://github.com/FuelLabs/sway-lib-std" }
```

Comment:
Rather than default branch, just leave this implicit. Then it'll use the stdlib version that matches the `forc` version.